### ISSUE TITLE
no-return-wrap doesn't work

### ISSUE THREAD

**Issue by trusktr** [xjamundx/eslint-plugin-promise#150]
I've got this code:

```js
		return p.then(function(val) {
			return Promise.resolve(val * 4)
		})
```

but ESLint does not show an error although I set

```js
		'promise/no-return-wrap': 'error',
```

v3.6.0

**Comment by trusktr** [comment]
Updated to v4.0.1, same behavior

**Comment by trusktr** [comment]
If I remove the `return`, then I do see the error, although the error is wrong because I'm not returning anything:

```js
		return p.then(function(val) {
			Promise.resolve(val * 4)
		})
```

**Comment by trusktr** [comment]
![screen shot 2018-10-25 at 5 37 58 pm](https://user-images.githubusercontent.com/297678/47537801-bcce9680-d87c-11e8-8b38-f9f912204e3d.png)

**Comment by tiagosiebler** [comment]
Any updates on this?

**Comment by xjamundx** [comment]
Not yet sorry, happy to take any pull requests
